package.json

```diff
@@ -28,6 +28,7 @@
     "query-string": "^9.1.1",
     "react": "^18.3.1",
     "react-app-polyfill": "^3.0.0",
+    "react-bootstrap": "^2.10.6",
```

Comment:
Hmm, we already had bootstrap, and tabs here https://github.com/gulfofmaine/Neracoos-1-Buoy-App/blob/main/app/(platformMap)/%40bottom/platform/%5BplatformId%5D/tabs.tsx using reactstrap

<img width="656" alt="image" src="https://github.com/user-attachments/assets/2e379dde-2b90-4efa-a197-0a47a88778af">

Reply:
Okay I got it working with reactstrap--just had to create a new reusable component that doesn't change the url and only changes the tab panels below the tabs.